core/src/test/scala/unit/kafka/server/BaseRequestTest.scala

```diff
@@ -39,19 +39,14 @@ abstract class BaseRequestTest extends KafkaServerTestHarness {
   protected def propertyOverrides(properties: Properties) {}
 
   def generateConfigs() = {
-    val props = TestUtils.createBrokerConfigs(numBrokers, zkConnect, enableControlledShutdown = false,
+    val props = TestUtils.createBrokerConfigs(numBrokers, zkConnect,
+      enableControlledShutdown = false, enableDeleteTopic = true,
       interBrokerSecurityProtocol = Some(securityProtocol),
       trustStoreFile = trustStoreFile, saslProperties = saslProperties)
     props.foreach(propertyOverrides)
     props.map(KafkaConfig.fromProps)
   }
 
-  @Before
-  override def setUp() {
-    super.setUp()
-    TestUtils.waitUntilTrue(() => servers.head.metadataCache.getAliveBrokers.size == numBrokers, "Wait for cache to update")
-  }
```

Comment:
This code was not actually required for any tests and actually caused the AuthorizerIntegrationTest to fail. 

It failed because this setup method is called before the AuthorizerIntegrationTest setup and therefore no messages are authorized to be sent in the cluster.